ietf-microwave-radio-link.yang

```diff
@@ -532,28 +514,16 @@ module ietf-microwave-radio-link {
             "Selected maximum coding/modulation.
              Adaptive coding/modulation shall not go
              above this value.";
+        }
+		leaf reference-modulation {
+          type identityref {
+            base mw-types:coding-modulation;
+          }
+          description
+            "reference-modulation is a reference to a coding-modulation value between the 'selected-min-acm' and the 'selected-max-acm.";
```

Comment:
The description is not fully clear to me. From https://github.com/samans/draft-ybam-rfc8561bis/issues/16#issuecomment-1541879715 I guess this attribute is the coding modulation to be used in the reference mode, as defined in ETSI EN 302 217-1 V3.2.2 (2020-02)

Please consider rephrasing and, if needed, adding the reference to ETSI EN 302 217-1 V3.2.2 (2020-02)